server/tserver/src/main/java/org/apache/accumulo/tserver/AssignmentHandler.java

```diff
@@ -285,6 +285,11 @@ public static boolean checkTabletMetadata(KeyExtent extent, TServerInstance inst
       return false;
     }
 
+    if (meta.getOperationId() != null) {
+      log.info(METADATA_ISSUE + "metadata entry has a FATE operation id");
```

Comment:
Can add some info to the logging

```suggestion
      log.info(METADATA_ISSUE + "metadata entry has a FATE operation id {} {} {}", extent, loc, meta.getOperationId());
```

Reply:
Implemented in 019cd50